include/AsyncTraceDispatcher.h

```diff
@@ -0,0 +1,118 @@
+#ifndef __AsyncTraceDispatcher_H__
```

Comment:
please use big alphabet for macro definition,  the same for following files
__AsyncTraceDispatcher_H__ change to __ASYNCTRACEDISPATCHER_H__